### ISSUE TITLE
Sphinx API warning / error

### ISSUE THREAD

**Issue by cdeil** [astropy/regions#39]
After merging #32 the Sphinx build on travis-ci now fails ... I think because of this warning:
```
/home/travis/build/astropy/regions/docs/api.rst:12: WARNING: toctree references unknown document u'api/regions.pixel_to_skycoord'
```
Full log: https://travis-ci.org/astropy/regions/jobs/139781170#L701

I'm not sure why that's happening or how to fix it.
@larrybradley @bsipocz - Help!?

**Comment by larrybradley** [comment]
@cdeil The `shape` modules were missing `__all__` statements.  I just submitted #40 to fix this.

**Comment by cdeil** [comment]
Fixed by @larrybradley in #40.

The builds on travis-ci and readthedocs are now clean:
https://travis-ci.org/astropy/regions/jobs/139800832
http://readthedocs.org/projects/astropy-regions/builds/4131540/
